Resources/Textures/Structures/Wallmounts/Lighting/emergency_light.rsi/meta.json

```diff
@@ -5,7 +5,12 @@
         "y": 32
     },
     "license": "CC-BY-SA-3.0",
+    "copyright": "juliang, modified by mirrorcult",
```

Comment:
@juliangiebel is this correct? thats as far back as the blame went and i guess this just didnt have copyright info before

Reply:
Yea I made that sprite 😄 
Nice PR! 👍